.github/workflows/typos.yml

```diff
@@ -1,4 +1,4 @@
-name: Check for typos
```

Comment:
I'd keep the more descriptive name

Reply:
<img width="379" alt="Screenshot 2024-04-29 at 10 20 32" src="https://github.com/taikoxyz/taiko-mono/assets/99078276/af3da3a3-bc12-4d05-8d42-6bd3b781b767">
The names are displayed in a sidebar, not sure if long names will fit.